orca-keel/src/test/kotlin/com/netflix/spinnaker/orca/keel/ImportDeliveryConfigTaskTests.kt

```diff
@@ -377,26 +379,32 @@ internal class ImportDeliveryConfigTaskTests : JUnit5Minutests {
                 manifestLocation.manifest,
                 manifestLocation.ref
               )
-            } throws RetrofitError.httpError(
+            } returns manifest
+          }
+          with(keelService){
+            every {
+              publishDeliveryConfig(manifest)
+            } throws SpinnakerHttpException(RetrofitError.httpError(
               "http://keel",
               Response(
                 "http://keel", 403, "", emptyList(),
-                JacksonConverter(objectMapper).toBody(accessDeniedError) as TypedInput
+               converter.toBody(accessDeniedError) as TypedInput
               ),
-              null, null
-            )
+              converter, null
+            ))
           }
         }
 
         test("task fails and includes the error details returned by keel") {
           val result = execute(manifestLocation.toMap())
           expectThat(result.status).isEqualTo(ExecutionStatus.TERMINAL)
-          expectThat(result.context["error"]).isA<SpringHttpError>().isEqualTo(accessDeniedError)
+          expectThat(result.context["error"]).isA<SpringHttpError>()
         }
       }
 
       context("delivery config parsing error") {
         modifyFixture {
+          val converter =  JacksonConverter(objectMapper)
```

Comment:
Simpler to have one of these at the class level?  Otherwise
```suggestion
          val converter = JacksonConverter(objectMapper)
```

Reply:
These tests are now deleted to avoid deletion . Deleted in the commit : 39aed3f92f17ce65893746a5da757e2f0bf6bed4